app/static/settings/translations.json

```
@@ -263,4 +263,37 @@
         "dark": "අඳුරු",
         "system": "පද්ධතිය"
     }
```

Needs a comma after the previous entry

```suggestion
    },
```